app/views/admin/enterprises/form/_business_details.html.haml

```diff
@@ -57,3 +57,17 @@
   .row{"data-controller": "updateinput"}
     = bf.hidden_field :_destroy, {"data-updateinput-target": "input"}
     = f.submit t(".reset_form"), {class: 'secondary', "data-action": "click->updateinput#update", "data-updateinput-value": "true"}
+
+.row
+  %fieldset.alpha.no-border-bottom
+    %legend= t('Invoice_item_sorting')
+  .three.columns.alpha
+    %label= t('.sort_items_by_supplier?')
+    %div{'ofn-with-tip' => t('.sort_items_by_supplier_tip')}
+      %a= t 'admin.whats_this'
+  .three.columns
+    = f.radio_button :preferred_invoice_order_by_supplier, true, 'ng-model' => 'Enterprise.preferred_invoice_order_by_supplier', 'ng-value' => 'true'
+    = f.label :enabled, t('.enabled'), value: :true
```

Comment:
These labels are not associated to the inputs. I think this may work:
```suggestion
    = f.label :preferred_invoice_order_by_supplier, t('.enabled'), value: :true
```